### PR TITLE
[DEV-721] basic setup

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+SQL_PORT=
+SQL_USERNAME=
+SQL_PASSWORD=
+SQL_DB=
+SQL_SYNCHRONIZE=
+SQL_MIGRATE=
+JWT_SECRET=
+MAIL_USER=
+MAIL_PASS=
+GH_TOKEN=
```
[the remark this answers is not in the document]
Comment (Krysh90) [reply .env.example:13]: if I understood correctly yesterday, that this github token is used for voting. If true, we will still need it

File: src/shared/auth/user-role.enum.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export enum UserRole {
+  // user roles
+  USER = 'User',
+  VIP = 'VIP',
+  BETA = 'Beta',
+  ADMIN = 'Admin',
+
+  // service roles
+  MASTERNODE_OPERATOR = 'MasternodeOperator',
+}
```
[the remark this answers is not in the document]
Comment (Krysh90) [reply src/shared/auth/user-role.enum.ts:10]: Agree, will be removed

File: src/shared/services/mail.service.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+interface KycMailContent {
+  salutation: string;
+  body: string;
+  subject: string;
+}
+
+@Injectable()
+export class MailService {
```
[the remark this answers is not in the document]
Comment (Krysh90) [reply src/shared/services/mail.service.ts:40]: okay - will be removed

File: .env.example (hunk 1 above)
[the remark this answers is not in the document]
Comment (davidleomay) [reply .env.example:13]: In DFX it was only used for the CFP API, to get all voting information from Github (for our public CFP API). It's not related to the DFX staking service.

[the remark this answers is not in the document]
Comment (Krysh90) [reply .env.example:13]: okay then I misunderstood that yesterday. thanks for clarifying. Will be removed

File: src/config/config.ts [hunk 4]
```diff
@@ -19,8 +17,6 @@ export class Configuration {
   defaultVolumeDecimal = 2;
   defaultPercentageDecimal = 2;
```
Comment (davidleomay) [inline src/config/config.ts:18]: The following config entries should IMO be unused and can be removed:
- githubToken = process.env.GH_TOKEN;
- defaultLanguage = 'en';
- defaultCountry = 'DE';
- defaultCurrency = 'EUR';
- defaultVolumeDecimal = 2;
- defaultPercentageDecimal = 2;

**Review by davidleomay — APPROVED** [review]

**Pull request merged**